homework3/semi-uber/Server/api/Schemas/load.schema.js

```diff
@@ -1,28 +1,26 @@
 const mongoose = require("mongoose");
 const Joi = require("@hapi/joi");
 const Schema = mongoose.Schema;
+const { STATUS } = require("../../data/loadData.json");
+const dimensionsSchema = require("./dimensions.schema");
 
 const Load = new Schema({
   created_by: { type: String },
   logs: { type: Array },
-  assigned_to: { type: String },
-  status: { type: String },
-  state: { type: String },
+  assigned_to: { type: String, default: "Not assigned" },
+  status: { type: String, default: STATUS.NEW },
+  state: { type: String, default: "Created" },
```

Comment:
Move this states to a constant, like you did with `STATUS`

Reply:
done